.gitignore

```diff
@@ -51,4 +51,14 @@ ircmaxell/password-compat/.travis.yml
 ircmaxell/password-compat/README.md
 ircmaxell/password-compat/phpunit.xml.dist
 ircmaxell/password-compat/test/
-ircmaxell/password-compat/version-test.php
\ No newline at end of file
+ircmaxell/password-compat/version-test.php
+
+ircmaxell/random-lib/test/
+ircmaxell/random-lib/.gitignore
+ircmaxell/random-lib/.travis.yml
+ircmaxell/random-lib/LICENSE
+ircmaxell/random-lib/README.md
+ircmaxell/random-lib/composer.json
```

Comment:
From what I can remember @MorrisJobke told me to also blacklist those files on some last PR here. I have no idea what is the right approach here :smile: